cmd/nodepool/aws/create.go

```diff
@@ -0,0 +1,114 @@
+package aws
```

Comment:
please update the docs getting starting guide to reflect this changes

Reply:
Updated the docs (added `aws` option)